wingtips-zipkin/src/main/java/com/nike/wingtips/zipkin/WingtipsToZipkinLifecycleListener.java

```diff
@@ -94,7 +99,14 @@ public void spanSampled(Span span) {
 
     @Override
     public void spanCompleted(Span span) {
-        zipkin.Span zipkinSpan = zipkinSpanConverter.convertWingtipsSpanToZipkinSpan(span, zipkinEndpoint, localComponentNamespace);
-        zipkinSpanSender.handleSpan(zipkinSpan);
+        try {
+            zipkin.Span zipkinSpan = zipkinSpanConverter.convertWingtipsSpanToZipkinSpan(span, zipkinEndpoint, localComponentNamespace);
+            zipkinSpanSender.handleSpan(zipkinSpan);
+        }
+        catch(Throwable ex) {
+            zipkinConversionOrReportingErrorLogger.warn("Found a span that is not zipkin compatible, or there was an error during span handling. "
```

Comment:
I see what you mean about being abuseable by malicious clients. I'll put in a counter and some safeties so it only logs once a minute.